docs/modules/ROOT/pages/type-definitions/schema-configuration/global-configuration.adoc

```diff
@@ -0,0 +1,51 @@
+[[schema-configuration-global-configuration]]
+= Global Configuration
+
+Through the schema configuration, it is possible to globally disable specific types of operation.
```

Comment:
This page makes it seem as though this is only possible to use `@query` in this way. What do you think of this wording?

```suggestion
Through the schema configuration, it is possible to globally disable specific types of operation using the `@query`, `@mutation` and `@subscription` directives.
```